### ISSUE TITLE
Allow to set image size in message

### ISSUE THREAD

**Issue by Batwam** [binwiederhier/ntfy#1068]
:bulb: **Idea**
As discussed on Matrix, it would be useful if there was a possibility to specify the size an the image attached to a message ( as a % of the window's width). Currently, images appear to be blow up to full width which in Firefox can we quite large. 

I did try to resize the picture hoping that it would show as a smaller thumbnail but it also gets scaled up to full message width which makes it very pixelated. As a minimum, I would propose (if possible) to only scale down but not up.

I also tried to include the picture using html markdown but this does not work as ntfy doesn't support html_markdowns. I don't believe that markdowns currently allow to adjust the picture size. Either way, I can't really use markdowns as they are shown as plain text on iOS (separate issue).

top picture resolution 160x66
bottom picture resolution 1080x475
actual message window width ~700px
![image](https://github.com/binwiederhier/ntfy/assets/56710655/cf6d3cf8-914a-4591-8236-64a61331d075)

:computer: **Target components**
Where should this feature/enhancement be added? 
Firefox/Android app as this is where images can currently be shown

**Comment by Batwam** [comment]
also, in the absence of markdown solution to embed pictures on iOS, I am having to use attachments to include pictures in the messages. However, I don't need these "Copy URL and OPEN Attachment" options. Would it be possible to have an option to remove those?

**Comment by wunter8** [comment]
That option has been discussed before. I'm not sure if it'll be implemented or of the best way to do that. However, in the meantime, if you use the PWA on iOS, markdown will work (including images)

**Comment by Batwam** [comment]
Yes, images work in the web browser and therefore PWA. This could be a solution but I am not receiving notifications when using PWA which makes this solution unusable for me. 

Unless notifications are meant to work with PWAs and I'm doing something wrong?

**Comment by wunter8** [comment]
Notifications can work with the PWA. If you're self hosting, you need to set up web push keys first: https://docs.ntfy.sh/config#web-push

**Comment by Batwam** [comment]
I am indeed self-hosting. I have now set it up in server.yml, however, the app is showing ""notifications are not supported in your browser" on firefox, chrome and safari so, no notifications for me :-(

**Comment by Batwam** [comment]
thanks for the info. I tried the PWA but it required setting up https to get notifications. I somehow never managed to make this work

**Comment by wunter8** [comment]
Yes, web push (notifications for the PWA) requires a domain name with a valid (e.g., not self-signed) TLS cert for HTTPS.

If you need help troubleshooting, Discord works a bit better than here
